src/components/picker/PickerDialog.android.js

```diff
@@ -83,7 +83,7 @@ class PickerDialog extends BaseComponent {
   render() {
     const dialogProps = extractComponentProps(Dialog, this.props);
```

Comment:
`this.props` already contains `panDirection` as null, passed from `NativePicker`.
For some reason `extractComponentProps` not populating it when spreading passed props

Reply:
You are right, I missed that. 
It should have already worked. 
Checking...

Reply:
Fixed (kinda)